tianshou/policy/modelfree/redq.py

```diff
@@ -153,7 +153,10 @@ def forward(  # type: ignore
         loc_scale, h = self.actor(batch.obs, state=state, info=batch.info)
         loc, scale = loc_scale
         dist = Independent(Normal(loc, scale), 1)
-        act = loc if self.deterministic_eval and not self.training else dist.rsample()
```

Comment:
I think we should extract a method out of this as it is used in multiple places. I would suggest a simple
`get_action()` method.

Reply:
Yes, you can add a method `get_action_from_dist(dist: Distribution)` to PGPolicy and SACPolicy. Pls add a docstring describing its behavior

Reply:
I guess you'll do it in the followup PR. Merging this